### ISSUE TITLE
Show reviews in full length when federating with other platforms

### ISSUE THREAD

**Issue by eszter007** [bookwyrm-social/bookwyrm#2573]
**Is your feature request related to a problem? Please describe.**
When I open my bookwyrm profile on Mastodon, I notice the following inconsistencies:
- Comments are shown in full length; the context (comment on {book title}) is shown in the end and is not localized
- Review texts are cropped; the context is shown in the beginning (review of {book title}) and is localized.

See screenshots.

**Describe the solution you'd like**
Comments and reviews should have a consistent behavior. 
The reviews are shown in full length (like the comments) without needing to open the website. Furthermore, the post format of a comment and review should look the same. Instead of showing (comment on {book title}) at the end of the post, it should be shown in the beginning:

```
Comment on {book title}:
{comment text}
```

Finally, the posts should also have a consistent localization behavior. Some posts are shown in English ("comment on...", "...finished reading...") while others are localized (e.g., reviews). 

**Describe alternatives you've considered**
Use the comment review format for all posts and crop the comments. I prefer not cropping the texts though since I think it shouldn't be necessary to open another website as it takes you out of the context. If the user wants to view the review on bookwyrm, they could just click on the Mastodon UI to view it on the origin instance.

**Additional context**
![image](https://user-images.githubusercontent.com/30467951/210635312-080952f3-2788-43b0-985a-556e3bff4965.png)
![image](https://user-images.githubusercontent.com/30467951/210635332-d03ebfd6-cf1b-48dd-a8b3-0e11b7f11641.png)

**Comment by hughrun** [comment]
I think this might be partially fixed by #2652 (pending code review) - but the localisation and consistency of headings still needs to be looked at.

**Comment by chdorner** [comment]
Not sure it will be fixed by #2652, that PR only has frontend fixes which shouldn't affect how statuses are federated and displayed on other systems like Mastodon

**Comment by mouse-reeve** [comment]
Yes, I think the fix here would be to use `Note`s for reviews, rather than `Article`s

**Comment by skolima** [comment]
related: https://github.com/bookwyrm-social/bookwyrm/issues/3412
